docs/data-integrations/amazon-redshift.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Amazon Redshift
+sidebarTitle: amazon-redshift.mdx
```

Comment:
Please replace `amazon-redshift.mdx` with `Amazon Redshift`.

It should look like this:
```
---
title: Amazon Redshift
sidebarTitle: Amazon Redshift
---
```